src/ci/citool/Cargo.toml

```diff
@@ -6,9 +6,11 @@ edition = "2021"
 [dependencies]
 anyhow = "1"
 clap = { version = "4.5", features = ["derive"] }
+csv = "1"
 serde = { version = "1", features = ["derive"] }
 serde_yaml = "0.9"
 serde_json = "1"
+ureq = { version = "3", features = ["json"] }
```

Comment:
interesting, I always used hyper or reqwest. Out of curiosity, are you using ureq because it's sync instead of async?

Reply:
I would also default to `reqwest`, but if you use it the dependency count here jumps to 150+, IIRC. We need this package to build fast on CI, and we don't really need async here, so it would just be wasteful. If we have another use-case for async stuff in the future, I would switch to `reqwest`.